src/modules/commander/Commander.cpp

```diff
@@ -3377,40 +3362,113 @@ void Commander::send_parachute_command()
 
 void Commander::checkWindSpeedThresholds()
 {
-	wind_s wind_estimate;
+	float max_allowed_wind_speed = _param_com_wind_max.get();
+#if PX4_EXPORT_RESTRICTION_BUILD
+
+	if (max_allowed_wind_speed > 12.f || max_allowed_wind_speed <= 0.f) {
```

Comment:
Maybe a comment for where the magic 12 comes from?

Reply:
or a named constant :)

Reply:
done